pangeo-notebook/conda-linux-64.lock

```diff
@@ -366,7 +358,7 @@ https://conda.anaconda.org/conda-forge/noarch/zict-2.2.0-pyhd8ed1ab_0.tar.bz2#cd
 https://conda.anaconda.org/conda-forge/noarch/aiosignal-1.2.0-pyhd8ed1ab_0.tar.bz2#68c9a0ab410c72abd24cbaac57703519
 https://conda.anaconda.org/conda-forge/linux-64/anyio-3.6.1-py39hf3d152e_0.tar.bz2#a1e0e704bc87ddb9be70192d2e4398d2
 https://conda.anaconda.org/conda-forge/linux-64/argon2-cffi-bindings-21.2.0-py39hb9d737c_2.tar.bz2#76139de3552a2046135eb0b2d02a9c85
-https://conda.anaconda.org/conda-forge/linux-64/arrow-cpp-8.0.0-py39heccc63a_1_cpu.tar.bz2#300ebd380aae1fe0bbf04ff1424b042e
+https://conda.anaconda.org/conda-forge/linux-64/arrow-cpp-6.0.1-py39h15c4769_19_cpu.tar.bz2#049c51026ed6a8a9a02a3ae27e07172e
```

Comment:
just noting seems like a big drop?

Reply:
huh! I wonder why they have this pinned to